cypress.json

```diff
@@ -4,5 +4,11 @@
   "reporter": "cypress-multi-reporters",
   "reporterOptions": {
     "configFile": "reporter-config.json"
+  },
+  "env": {
+    "openSearchUrl": "http://localhost",
+    "openSearchPort": 9200,
```

Comment:
we need to change cypress.json file in order to pick up the test settings. could we make it part of system environment variable or input of launch script parameter, so we don't need to change file for different environment

Reply:
the cypress environment variable can be override by CLI at execution time, ref: https://docs.cypress.io/guides/references/configuration#Environment-Variables